webapp/src/components/modals/channel_settings/select_channel_subscription.tsx

```diff
@@ -49,15 +49,15 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
                     </button>
                 </div>
                 {errorDisplay}
-                {this.props.channelSubscriptions.map((sub) => (
-                    <table className='table'>
-                        <thead>
-                            <tr>
-                                <th scope='col'>ID</th>
-                                <th scope='col'>Actions</th>
-                            </tr>
-                        </thead>
-                        <tbody>
+                <table className='table'>
+                    <thead>
+                        <tr>
+                            <th scope='col'>{'Name'}</th>
+                            <th scope='col'>{'Actions'}</th>
+                        </tr>
+                    </thead>
+                    {this.props.channelSubscriptions.map((sub, i) => (
+                        <tbody key={i}>
```

Comment:
There should be just one `tbody` right? I'm thinking the `map` should be the next level down, wrapping the `td` element.

Reply:
Correct!  Originally, the `map` was creating several tables too.  I've restructured in the latest edit so the that there is one `<table>`, `<thead>`, `<tbody>` and each subscription is now a `<tr>` element.

Reply:
Correct!  Originally, the `map` was creating several tables too.  I've restructured in the latest edit so there is one `<table>`, `<thead>`, `<tbody>` and each subscription is now a `<tr>` element.

Reply:
Great fix :+1: